spec/memoizable/dependency_spec.rb

```diff
@@ -155,6 +155,44 @@ def calc_b_c(x)
           5.times { obj.calc(val) }
         end
       end
+
+      context 'for different parameter formats in the dependency block' do
+        it 'works using a dependency block with a splat' do
+          obj.class_eval do
+            def test(*args, **kwargs); end
+            memoize_method :test do |_, *args|; end
+          end
+          RedisMemo::Cache.with_local_cache do
+            expect(RedisMemo::MemoizeMethod).to receive(:extract_dependencies).thrice.and_call_original
```

Comment:
maybe we can wrap around each sub-testcase and expect the method to be called once?

for example:
```ruby
before { add_method_test }

add_test_case. { 5.times { obj.test(1) } }
add_test_case. { 5.times { obj.test(1, 1, 1) } }
```